lib/FusionInventory/Agent/Task/Inventory/MacOS/CPU.pm

```diff
@@ -53,7 +53,8 @@ sub _getCpus {
     }
     close $handle;
 
-    my $type  = $sysprofile_info->{'Processor Name'} ||
+    my $type  = $sysctl_info->{'machdep.cpu.brand_string'} || 
```

Comment:
I merged and I'll apply the requested change directly.
Thank you for your contribution.